tlspuffin/src/integration_tests/mutations.rs

```diff
@@ -46,133 +46,167 @@ fn test_mutate_seed_cve_2021_3449() {
     let mut state = create_state();
     let _server = AgentName::first();
 
-    expect_crash(move || {
-        for _i in 0..5 {
-            let mut attempts = 0;
+    forked_execution(
+        move || {
+            for _i in 0..5 {
+                let mut attempts = 0;
 
-            let (mut trace, _) = _seed_client_attacker12(AgentName::first());
+                let (mut trace, _) = _seed_client_attacker12(AgentName::first());
 
-            // Check if we can append another encrypted message
+                // Check if we can append another encrypted message
 
-            let mut mutator = RepeatMutator::new(15);
+                let mut mutator = RepeatMutator::new(15);
 
-            fn check_is_encrypt12(step: &Step<TlsQueryMatcher>) -> bool {
-                if let Action::Input(input) = &step.action {
-                    if input.recipe.name() == fn_encrypt12.name() {
-                        return true;
+                fn check_is_encrypt12(step: &Step<TlsQueryMatcher>) -> bool {
+                    if let Action::Input(input) = &step.action {
+                        if input.recipe.name() == fn_encrypt12.name() {
+                            return true;
+                        }
                     }
+                    false
```

Comment:
@michaelmera just so that you know, these tests here were an effort during the last weeks of paper writing to find reasons for fuzzing limitations. So they are low quality and likely should go away in the future.